io-engine/src/bdev/nvmx/channel.rs

```diff
@@ -489,7 +489,6 @@ impl NvmeControllerIoChannel {
 
 impl Drop for NvmeControllerIoChannel {
     fn drop(&mut self) {
-        trace!("I/O channel {:p} dropped", self.0.as_ptr());
```

Comment:
Why is this removed, seems useful to have? Or is it too verbose?

Reply:
This looks like a mistake after I had modified these traces during testing. Will put back.